internal/backend/builds/builds.go

```diff
@@ -56,3 +63,21 @@ func (b *Builds) Save(ctx context.Context, build sdktypes.Build, data []byte) (s
 func (b *Builds) Delete(ctx context.Context, id sdktypes.BuildID) error {
 	return b.DB.DeleteBuild(ctx, id)
 }
+
+func (b *Builds) Describe(ctx context.Context, bid sdktypes.BuildID) (*sdkbuildfile.BuildFile, error) {
+	r, err := b.Download(ctx, bid)
+	if err != nil {
+		return nil, fmt.Errorf("download: %w", err)
+	}
+
+	defer r.Close()
```

Comment:
`defer` should be as close as possible to the thing it's attached to.
```suggestion
	}
	defer r.Close()
```

Reply:
fixed